sybil/parsers/abstract/skip.py

```diff
@@ -18,9 +18,9 @@ class AbstractSkipParser:
         :class:`~sybil.Document` that is parsed.
     """
 
-    def __init__(self, lexers: Sequence[Lexer]):
+    def __init__(self, lexers: Sequence[Lexer], directive: str):
         self.lexers = LexerCollection(lexers)
-        self.skipper = Skipper()
+        self.skipper = Skipper(directive=directive)
 
     def __call__(self, document: Document) -> Iterable[Region]:
```

Comment:
On line 30 there is a `get` default which should probably be replaced with the given `directive`. However, I didn't want to touch that as I don't understand when it gets used and no tests fail if I replace it with `'foobar'`.

Reply:
I agree that this should be `self.directive`, let me see if I can land a test on master that illustrates this.